src/blockchain/txpool.ts

```diff
@@ -97,6 +112,7 @@ export class TxPool {
       },
     })
 
+    head.pushStorageLayer().set(compactHex(meta.query.randomness.notFirstBlock()), StorageValueKind.Deleted)
```

Comment:
@xlc not sure what we could do here. Ideally, the signature should be mocked too but quickly looking at the randomness pallet I don't see much how this could be done.
A temporary solution is to remove the NotFirstBlock but it needs to be pallet specific and also break support for randomness.
Any idea ?

Reply:
how does the randomness pallet verify the VRF? if that's using host function than we can mock it. otherwise I can see few options:
- modify randomness pallet to make it mock friendly. e.g. if a particular storage exists or not exists, switch to mock mode
- refactor chopsticks to support plugins so we can better manage pallet specific code

Reply:
So it checks the NotFirstBlock there https://github.com/PureStake/moonbeam/blob/master/pallets/randomness/src/lib.rs#L294
And the verification is (few lines under) pointing to https://github.com/PureStake/moonbeam/blob/master/pallets/randomness/src/vrf.rs#L32